scripts/profiler/stream.py

```diff
@@ -3,58 +3,45 @@
 #
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 
-import socket
-import errno
-
 class StreamError(Exception):
     pass
 
 class Stream():
     RECV_BUF_SIZE = 2**13
 
-    def __init__(self, own_socket_dict, timeouts, remote_socket_dict=None):
-        self.own_sock_desc = socket.socket(socket.AF_UNIX, socket.SOCK_DGRAM)
-        self.own_sock_desc.bind(own_socket_dict['descriptions'])
-        self.own_sock_desc.settimeout(timeouts['descriptions'])
-        self.own_sock_ev = socket.socket(socket.AF_UNIX, socket.SOCK_DGRAM)
-        self.own_sock_ev.bind(own_socket_dict['events'])
-        self.own_sock_ev.settimeout(timeouts['events'])
-
-        self.remote_socket_dict = remote_socket_dict
+    def __init__(self, pipe_recv_desc=None, pipe_recv_ev=None, timeouts=None,
```

Comment:
Should we explicitly close the pipes as soon as they are not used?

Reply:
Pipes are closed automatically when they are garbage collected,